lib/kitchen/directions/book_answer_key_container/v1.rb

```diff
@@ -3,11 +3,11 @@
 module Kitchen::Directions::BookAnswerKeyContainer
   class V1
     renderable
-
-    def bake(book:)
+    def bake(book:, klass: 'solutions')
```

Comment:
Because you're setting the value of `klass` in main, I think you can remove here and just have `klass:`. 
I was doing a [similar thing in another PR](https://github.com/openstax/kitchen/pull/255/files) and Mary Jo gave me the pointer!

Reply:
Thank you! I wasn't sure, so I left it there too. :sweat_smile: